#SPDX-License-Identifier: CC0
diff --git a/debian/changelog b/debian/changelog
index c4dd05f..4b74096 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-xserver-xorg-input-mouse (1:1.9.3-2) UNRELEASED; urgency=medium
+xserver-xlibre-input-mouse (1:1.9.3-2) UNRELEASED; urgency=medium
 
   * control: Migrate to x11proto-dev.
 
diff --git a/debian/control b/debian/control
index e335e7d..c0aa5d1 100644
--- a/debian/control
+++ b/debian/control
@@ -1,4 +1,4 @@
-Source: xserver-xorg-input-mouse
+Source: xserver-xlibre-input-mouse
 Section: x11
 Priority: optional
 Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
@@ -14,7 +14,7 @@ Standards-Version: 3.9.8
 Vcs-Git: https://salsa.debian.org/xorg-team/driver/xserver-xorg-input-mouse.git
 Vcs-Browser: https://salsa.debian.org/xorg-team/driver/xserver-xorg-input-mouse
 
-Package: xserver-xorg-input-mouse
+Package: xserver-xlibre-input-mouse
 Architecture: any
 Depends:
  ${shlibs:Depends},
@@ -36,7 +36,7 @@ Description: X.Org X server -- mouse input driver
  .
  This package is built from the X.org xf86-input-mouse driver module.
 
-Package: xserver-xorg-input-mouse-udeb
+Package: xserver-xlibre-input-mouse-udeb
 XC-Package-Type: udeb
 Section: debian-installer
 Architecture: hurd-any kfreebsd-any
diff --git a/debian/xserver-xorg-input-mouse-udeb.install b/debian/xserver-xlibre-input-mouse-udeb.install
similarity index 100%
rename from debian/xserver-xorg-input-mouse-udeb.install
rename to debian/xserver-xlibre-input-mouse-udeb.install
diff --git a/debian/xserver-xorg-input-mouse.docs b/debian/xserver-xlibre-input-mouse.docs
similarity index 100%
rename from debian/xserver-xorg-input-mouse.docs
rename to debian/xserver-xlibre-input-mouse.docs
diff --git a/debian/xserver-xorg-input-mouse.install b/debian/xserver-xlibre-input-mouse.install
similarity index 100%
rename from debian/xserver-xorg-input-mouse.install
rename to debian/xserver-xlibre-input-mouse.install
diff --git a/debian/xserver-xorg-input-mouse.links b/debian/xserver-xlibre-input-mouse.links
similarity index 100%
rename from debian/xserver-xorg-input-mouse.links
rename to debian/xserver-xlibre-input-mouse.links
